### PR TITLE
Add default accessurl for my account and console

### PR DESCRIPTION
**Pull request by sandushi**
### Purpose
Auto login is not working in self registration flow. Resolved the issue by adding default access url for my account and console applications.

### Related Issues
- Resolved 
https://github.com/wso2/product-is/issues/13426

### CODE REVIEW

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java [hunk 1]
```diff
@@ -161,6 +161,58 @@ public static void createApplication(String appName, String appOwner, String app
...
+     * @param appOwner       Application owner.
+     * @param appDescription Application description.
+     * @param consumerKey    Consumer key.
+     * @param consumerSecret Consumer secret.
+     * @param portalPath     Portal path.
+     * @throws IdentityApplicationManagementException IdentityApplicationManagementException.
+     */
+    public static void createApplication(String appName, String appOwner, String appDescription, String consumerKey,
+                                         String consumerSecret, String tenantDomain, String portalPath)
+            throws IdentityApplicationManagementException {
```
Comment (piraveena) [inline components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:177]: can you remove the body of the previous `createApplication(String appName, String appOwner, String appDescription, String consumerKey,
            String consumerSecret, String tenantDomain)` and call the new method?

having two methods with similar logic introduces redundancy

Comment (IsuraD) [reply components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:145]: +1 Call the overloaded method internally

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java [hunk 2]
```diff
@@ -124,12 +125,33 @@ public static void createOAuth2Application(String applicationName, String portal
      */
     public static void createApplication(String appName, String appOwner, String appDescription, String consumerKey,
             String consumerSecret, String tenantDomain) throws IdentityApplicationManagementException {
+        createApplication(appName, appOwner, appDescription,
```
Comment (piraveena) [inline components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:128]: formatting issue

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java (hunk 1 above)
Comment (sandushi) [reply components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:145]: resolved

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java (hunk 2 above)
Comment (sandushi) [reply components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:129]: resolved

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java [hunk 3]
```diff
@@ -18,6 +18,7 @@
 
 package org.wso2.identity.apps.common.util;
 
+import org.apache.commons.lang3.StringUtils;
```
Comment (piraveena) [inline components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:21]: use `org.apache.commons.lang.StringUtils`

**Review by piraveena — APPROVED** [review]

File: components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java (hunk 1 above)
Comment (senthalan) [reply components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:146]: let's add a deprecated annotation for the method `createApplication(String appName, String appOwner, String appDescription, String consumerKey, String consumerSecret, String tenantDomain)`

Comment (sandushi) [reply components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java:148]: resolved

**Pull request merged**